### DISCUSSION TITLE
Bug Student Developper Pack

### DISCUSSION THREAD

**Discussion by martyduniaud98** [community/community#36253] [GitHub Education] [answered]
Hello,
I'm trying to activate my student developper pack but I can't find how, I'm well connected to github, I can enter my school in the form, my email is verified but it's not offered to me in the form and I can't enter email manually either, if anyone knows how to solve this problem?

I put you two small screen.

![formulaire](https://user-images.githubusercontent.com/92717628/195999519-67c559af-8363-4a86-9257-e413d7819278.JPG)
![git](https://user-images.githubusercontent.com/92717628/195999524-e57d7a81-ea2e-474c-9eac-be628defa118.JPG)

**Comment by ettaboyle** [comment, marked as answer]
Thanks for participating!

We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

You can find all the details you need about the GitHub Student Developer Pack in this post:
[Student Developer Pack Application & FAQs ➡️ Start Here](https://github.com/orgs/community/discussions/17814)

Please review all the information in that post and follow the steps outlined. Thank you!

**Reply by martyduniaud98** [reply]
I changed the language of my question, can you please unblock my post now

**Comment by nurahmed123** [comment]
![Screenshot_20221019_012700](https://user-images.githubusercontent.com/84973115/196525611-0c8e7737-035a-4c6f-a1c5-0697ed0fba79.jpg)
I claimed the package and on my Github profile pro badge showing but i couldn't access any of my students package benefits. How may i fixed this issue
